recipes/pytest-localstack/meta.yaml

```diff
@@ -0,0 +1,52 @@
+{% set name = "pytest-localstack" %}
+{% set version = "0.4.1" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: 91be109a98f3737e10b0b1de8f0cf48f30abee878a77099862fad2bed6de689c
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - pip
+    - python
+  run:
+    - botocore >=1.4.31,!=1.4.45
+    - contextlib2  # [ py < 33 ]
+    - docker-py
+    - mock  # [ py < 33 ]
+    - pluggy >=0.6.0
+    - pytest >=3.3.0
+    - python
+    - setuptools
+    - six
+
+test:
+  imports:
+    - pytest_localstack
+    - pytest_localstack.contrib
+  requires:
+    - boto3
+    - hypothesis
```

Comment:
I think the ``requires`` here are not necessary, because as you are just testing if the library is being imported, you will not need those libraries.